### PR TITLE
[ticket/8116] Add pagination for IP tables and post info

### PR DESCRIPTION
**Pull request by marc1706**
Checklist:

- [x] Correct branch: master for new features; 3.2.x, 3.1.x for fixes
- [x] Tests pass
- [x] Code follows coding guidelines: [master / 3.2.x](https://area51.phpbb.com/docs/master/coding-guidelines.html), [3.1.x](https://area51.phpbb.com/docs/31x/coding-guidelines.html)
- [x] Commit follows commit message [format](https://wiki.phpbb.com/Git#Commit_Messages)

Tracker ticket (set the ticket ID to **your ticket ID**):

https://tracker.phpbb.com/browse/PHPBB3-8116

### CODE REVIEW

**Comment by hanakin** [comment]
not sure whats going on here as ticket 8116 is not this and the link to tracker is default also not this?

anyway...no <> icons?

**Comment by marc1706** [comment]
This is the PR for this ticket: https://tracker.phpbb.com/browse/PHPBB3-8116
Also take a look at the previous PR by nickvergessen: https://github.com/phpbb/phpbb/pull/3456
Couldn't cherry-pick his previous commit because he deleted the branch the PR was based on.

**Comment by hanakin** [comment]
ah i see its 3.1 damn ok so new prs for 3.2.x and 3.3 to handle icons yeah

**Comment by Nicofuma** [comment]
ping @marc1706

**Comment by marc1706** [comment]
@Nicofuma fixed and added PR for 3.2.x

File: tests/mcp/post_ip_test.php [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+	}
+
+	/**
+	 * @dataProvider data_get_num_posters
+	 */
+	public function test_get_num_posters($expected, $ip)
+	{
+		$this->assertSame($expected, phpbb_get_num_posters_for_ip($this->db, $ip));
+	}
+}
```
Comment (Nicofuma) [inline tests/mcp/post_ip_test.php]: missing line :)

Comment (marc1706) [inline tests/mcp/post_ip_test.php]: Argh

**Comment by marc1706** [comment]
Updated & also updated 3.2.x PR

**Comment by marc1706** [comment]
Fixed both this and the 3.2.x PR @Nicofuma

**Pull request merged**